client/js/app.modules/main/main.controller.js

```diff
@@ -0,0 +1,7 @@
+/**
```

Comment:
In our case we started to use Menu (like navigation) and Main (like menu) names since were created the first mock-ups and scss styles. So I continued this "logic".